lib/src/sai_redis_generic_create.cpp

```diff
@@ -197,6 +198,94 @@ void redis_free_virtual_object_id(
     }
 }
 
+// wait for response
+sai_status_t redis_get_response(
+    _In_ const std::string &key,
+    _In_ const std::string &op,
+    _In_ sai_object_type_t object_type)
+{
+    SWSS_LOG_ENTER();
+
+    swss::Select s;
+
+    sai_status_t status;
+    std::string request;
```

Comment:
How request is used here? I see it's used for output only. I don't see any assignment here.